File: test/golden/request/ruby/1.rb
# Built by LucyBot. www.lucybot.com
uri = URI("https://api.foo.com/bar/baz")
http = Net::HTTP.new(uri.host, uri.port)
http.use_ssl = true
headers = {
  "Content-Type" => "application/json",
  "apikey" => "asdf"
}
uri.query = URI.encode_www_form({
  "q" => "Obama"
})
request = Net::HTTP::Post.new(uri.request_uri, headers)
request.body = [{
    "a" => 1,
    "b" => 2
  }, {
    "a" => 3,
    "b" => 4
  }].to_json
@result = http.request(request).body
puts @result

File: test/golden/request/node/1.js
// Built by LucyBot. www.lucybot.com
request.post({
  url: "https://api.foo.com/bar/baz",
  qs: {
    'q': "Obama"
  },
  headers: {
    'Content-Type': "application/json",
    'apikey': "asdf"
  },
  body: JSON.stringify([{
      'a': 1,
      'b': 2
    }, {
      'a': 3,
      'b': 4
    }]),
}, function(err, response, body) {
  console.log(body);
})
